zerobuf/Zerobuf.h

```diff
@@ -47,6 +47,17 @@ class Zerobuf
     /** @return the size of the zerobuf. */
     ZEROBUF_API size_t getZerobufSize() const;
```

Comment:
If there is no function named getSize() I wonder if "Zerobuf" is needed in getZerobufSize() function name